aspnetcore/mvc/views/tag-helpers/builtin/resources/CacheTagHelper.md

```diff
@@ -0,0 +1,286 @@
+﻿[Back To Built In Tag Helpers List](../../builtin.md)
+
+# Cache Tag Helper
+
+By [Peter Kellner](http://peterkellner.net) 
+
+
+The  Cache Tag Helper provides the ability to dramatically improve the performance of your ASP.NET Core app by caching its content to the internal ASP.NET Core cache provider.
+
+A simple example that shows the Cache Tag Helper in action sets the current time in its content area. The Razor View Engine sets the default `expires-after` to twenty minutes, which is the default if you specify no additional attributes.
+
+Here is an example of the code you include in your `cshtml` page to achieve the default cache behavior described above.
+
+```html
+<Cache>@DateTime.Now<Cache>
+```
+
+If you repeatedly refresh the page that contains `CacheTagHelper`, the first time the current`DateTime`will be shown. Repeated refreshes of the page will continue to show the first `DateTime` shown until the cache expires (which is likely to be twenty minutes unless server memory pressure evicts the cache sooner).
+
+You can achieve more control of the cache duration of the content by setting any of the following attributes:
+
+## Cache Tag Helper Attributes
+
+- - -
+
+### enabled    
+
+
+| Attribute Type 	| Valid Values   	|
+|----------------	|----------------	|
+| boolean         	| "true" (default) 	|
+|               	| "false"	|
+
+
+The attribute `enabled` determines whether any caching is performed of the content inside the Cache Tag Helper. The default is `true`, which results in all other attributes being valid.  The default value is `true`.  If set to `false`, regardless of what other attributes are specified, this Cache Tag Helper will have no caching effect on the rendered output.
```

Comment:
* Go with one space between sentences
* "The default is `true`" and "The default value is `true`" ... so this information appears twice
* The last sentence has a "will," and the verb form is "affect."
* I may have had you add that comma after `false`, but I'm feel'in an "and" there might be good.

Here's an alternate version ...

If set to `false` and regardless of what other attributes are specified, this Cache Tag Helper has no caching affect on the rendered output.